handbook/engineering/README.md

```diff
@@ -772,30 +772,27 @@ For unreleased bugs in an active sprint, no bug report issue is created. Instead
 #### Bug States
 The lifecycle stages of a bug at Fleet are: 
 1. Inbox 
-2. Acknowledged 
-3. Reproduced 
-4. In engineering process
-5. Awaiting QA
+2. Reproduced 
+3. In engineering process
+4. Awaiting QA
 
 The above are all the possible states for a bug as envisioned in this process. These states each correspond to a set of GitHub labels, assignees, and board memberships. 
 
 See [Bug states and filters](#bug-states-and-filters) at the end of this document for descriptions of these states and links to each GitHub filter.
 
 #### Inbox
 When a new bug is created using the [bug report form](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=bug%2C%3Areproduce&template=bug-report.md&title=), it is in the "inbox" state. 
-At this state, the [bug review DRI](#rituals) (QA) is responsible for going through the inbox and asking for more reproduction details from the reporter, asking the product team for more guidance, or acknowledging the bugs.
 
-#### Weekly bug review
-QA has weekly check-in with product to go over the inbox items. QA is responsible for proposing “not a bug”, closing due to lack of response (with a nice message), or raising other relevant questions. All requires product agreement
+At this state, the [bug review DRI](#rituals) (QA) is responsible for going through the inbox and documenting reproduction steps, asking for more reproduction details from the reporter, or asking the product team for more guidance. QA has one week to move the bug to the next step (reproduced).
 
-QA may also propose that a reported bug is not actually a bug. A bug is defined as “behavior that is not according to spec or implied by spec.” If agreed that it is not a bug, then it's assigned to the relevant product manager to determine its priority.
+For community-reported bugs, this may require QA to gather more information from the reporter. QA should reach out to the reporter if more information is needed to reproduce the issue. Reporters have six weeks to provide follow-up information for each report. We'll ping them again as a reminder at three weeks. After six weeks, we'll close the bug to remove it from our visibility, but reporters are welcome to re-open and provide context.
 
-#### Acknowledging bugs
-If the inbox item is a bug, QA should apply the acknowledged state to the bug. QA has one week to reproduce the bug.
+Once reproduced, QA should document the reproduction steps in the description, align with product on what is the desired outcome during the weekly bug review meetings, and move it to the reproduced state.
```

Comment:
My concern here is bugs being delayed because they are waiting on the weekly review. Instead, we should take action on bugs immediately. How about something like this?